internal/integration/talos_test.go

```diff
@@ -318,6 +318,7 @@ func AssertEtcdMembershipMatchesOmniResources(testCtx context.Context, client *c
 		require.NoError(err)
 
 		err = retry.Constant(time.Minute*2, retry.WithUnits(time.Second)).Retry(func() error {
+			resp, _ = c.EtcdMemberList(ctx, &machine.EtcdMemberListRequest{})
```

Comment:
Should we swallow errors here? Maybe do retry.ExpectedError instead (only for expected types ofc, checking via errors.Is and/or GRPC's status.Code etc.). Otherwise hard fail.

Reply:
We already check error for this on the first call and fail fast if it returns error. I assumed that this error check can be omitted on subsequent calls.

Reply:
Agree, just also be careful with Eventually* stuff with context cancellation - check if context has error on each iteration, and fail fast if there is.